Build/Shared.build.props

```diff
@@ -58,6 +58,7 @@
 
   <ItemGroup>
     <PackageReference Include="Remotion.Infrastructure.Styles.Analyzer" Version="1.1.3" PrivateAssets="All" />
+    <PackageReference Include="Remotion.Core.Development.Analyzers" Version="1.0.0" PrivateAssets="All" />
```

Comment:
@MichaelKetting Deine Meinung? :)

Reply:
ProjectReference is correct for unit tests. Within the same solution we should never have PackageReferences because it would be...weird and likely not work/behave strangely. (see the separate analyzers repository)

As for referencing in library projects, puh, if we want this guard to also check things in re-motion, I don't know how things should work. The question is, do we want/need this guard also inside remotion itself? If so, we probably should reference an earlier, already released version to avoid blowing us up during development. Of course, there's also far less risk in re-motion, I'd hope. We could probably make this a separate issue from unit testing the project? Also, how do we handle the web analyzers @a-ctor ?

@Frazerus

Reply:
Using it in re-motion is not a problem. We do this with the Web.Analyzers already. The question is just which projects should receive the analyzer reference. If we do it in `Shared.build.props` everyone gets it, if we do it in a project type file only certain projects get the reference. I think it is fine to leave as is in `Shared.build.props`